src/useFeedbackFish.ts

```diff
@@ -10,10 +15,4 @@ export const useFeedbackFish = (projectId: string) =>
     script.addEventListener("error", onScriptError)
 
     document.body.appendChild(script)
-
-    return () => {
-      script.removeEventListener("error", onScriptError)
-
-      script.remove()
-    }
```

Comment:
Deleting this means that the eventListener and the script tag will still be in the DOM

Reply:
The JavaScript will continue running anyway, even if we remove the script tag (which caused bugs for many folks), hence I feel like it's more obvious to let it stick around so folks know the JS is still around :wink:

Reply:
Sounds like a good idea indeed 🙂